CONTRIBUTING.md

```diff
@@ -125,3 +125,33 @@ cd docs/_build/html/
 # Open the index file to view it in a browser:
 open _build/html/index.html
 ```
+
+
+## Release process
+
+### Editing the Code
+
+* Your changes should be on a branch.
+* Your changes should have unit and/or integration tests, depending on what the changes are for.
+* Your changes should pass the linter via the pre-commit hook/on Github.
+* Your code should pass all the unit tests (locally and on Github).
+* Get a full code review.
+* Push to Github and merge to `master`.
+
+### Release
+
+1. Run integration tests on `master`, locally or using [Github Actions](https://github.com/XRPLF/xrpl-py/actions/workflows/integration_test.yml).
```

Comment:
not locally, because we want to test all versions and most likely people aren't set up with all three versions locally. i'm not, for example.